### ISSUE TITLE
Audio recorder not visible in 0.0.9

### ISSUE THREAD

**Issue by jelmers19** [Joooohan/audio-recorder-streamlit#18]
In my existing code, i run the snippet below, which works fine in 0.0.8. Since i upgraded to 0.0.9, the icon and text are not visible, unless i manually perform a different action in my app (like toggling an unrelated radiobutton). Downgrading back to 0.0.8 solves the problem.

```
st.session_state.audio_bytes = audio_recorder(text="Click on mic to start recording",
                                                          pause_threshold=1.0,
                                                          key=st.session_state.key_recorder,
                                                          sample_rate=rec_sr,
                                                          )
```

**Comment by lkdd-ao** [comment]
I won't solve this problem by fixing the original code, but you can try using the following method 👇
```
import streamlit as st

st.html(
    '''
     <style>
        iframe[title="audio_recorder_streamlit.audio_recorder"] {
            height: auto;
        }
     </style>
    '''
)
```

**Comment by Joooohan** [comment]
Ok, I found the issue I will release it in the next version.

**Comment by Joooohan** [comment]
Should be fixed with version 0.0.10.
